statistics/table.go

```diff
@@ -464,19 +464,57 @@ func outOfRangeEQSelectivity(ndv, modifyRows, totalRows int64) float64 {
 	return selectivity
 }
 
+//
+func (coll *HistColl) crossValidate(sc *stmtctx.StatementContext, idx *Index, usedColsLen int, idxPointRange *ranger.Range) (float64, float64, error) {
+	minRowCount := math.MaxFloat64
+	cols := coll.Idx2ColumnIDs[idx.ID]
+	crossValidationCount := 1.0
```

Comment:
How about renaming it to `crossValidationSelectivity`?

Reply:
This comment has not been addressed?

Reply:
I don't understand. I think I have already modified the function name to `crossValidationSelectivity`.